config/codeceptjs_helpers.js

```diff
@@ -65,60 +65,56 @@ class customHelpers extends Helper {
   // Evaluates true if xpath is checked
   async seeCheckedAttribute (xpath) {
     const helper = this.helpers['Puppeteer'] || this.helpers['WebDriver']
-    let res = await helper.grabAttributeFrom(xpath, 'checked')
-    return assert.ok(res !== null && typeof res !== 'undefined' && res.toString().toLowerCase() === 'true', "\x1b[31mexpected element '\x1b[91m" + xpath + "\x1b[31m' to be checked")
+    let res = await helper.grabAttributeFromAll(xpath, 'checked')
+    return assert.ok(Array.isArray(res) && res.length > 0 && res[0].toString().toLowerCase() === 'true', "\x1b[31mexpected element '\x1b[91m" + xpath + "\x1b[31m' to be checked")
```

Comment:
In CodeceptJS 3, grabAttributeFrom() on a single element throws an error if the element is not found. This is a BC break documented in https://github.com/codeceptjs/CodeceptJS/wiki/Upgrading-to-CodeceptJS-3.

Switching to grabAttributeFromAll(), we can check for an empty error if the element is not found.